docs/vereisten/index.md

```diff
@@ -1,69 +1,88 @@
 ---
 title: Vereisten
+summary: Overzicht van eisen voor overheden die algoritmes of AI ontwikkelen of gebruiken.
 hide:
 - navigation
 ---
+# Eisen
+Overzicht van eisen voor overheden die algoritmes of AI-systemen ontwikkelen of gebruiken. Deze eisen zijn gebaseerd op de wetten en regels voor het uitvoeren van wettelijke taken, zoals de Grondwet.
```

Comment:
Misschien goed om hier een korte opsomming te geven van de wet- en regelgeving die is meegenomen?

- AI-verordening
- AVG
- Algemene Wet Bestuursrecht
- Auteurswet
- Europese verdragen
- Baseline Informatiebeveiliging Overheid

(misschien nog even checken of dit lijstje compleet is) 

Dit geeft de lezer volgens mij wel gelijk een idee wat hij/zij kan verwachten.

Reply:
En de [Algemene wet gelijke behandeling](https://wetten.overheid.nl/jci1.3:c:BWBR0006502)

Reply:
Extra voorbeelden toegevoegd.